Docs/Book/C++Examples/example8.cpp

```diff
@@ -0,0 +1,21 @@
+//
+// Modifying molecules example8.cpp
+
+#include <iostream>
+
+#include <GraphMol/GraphMol.h>
+#include <GraphMol/SmilesParse/SmilesParse.h>
+#include <GraphMol/MolOps.h>
+
+int main( int argc , char **argv ) {
+
+  RDKit::ROMOL_SPTR mol1( RDKit::SmilesToMol( "CCO" ) );
```

Comment:
You do it elsewhere too, but I like this use of smart pointers to avoid the memory leak problem and to make it explicit that the memory is owned here.